### DISCUSSION TITLE
GAME COMMENTS

### DISCUSSION THREAD

**Discussion by giscus** [CanPixel/BadOptics#2] [General]
# The Gamified Political Compass | THE GAMIFIED POLITICAL COMPASS - You know what they say, opposites repel.

THE GAMIFIED POLITICAL COMPASS - You know what they say, opposites repel.

https://canpixel.github.io/BadOptics/

**Comment by CanPixel** [comment]
Governing a country is hard, there is <i>no one way</i>. 
<b>Go figure it out!</b>

**Reply by Fengsemul** [reply]
Way 1: Press always "Agree" with Auto-Clicker. There's an authoritarian-right bias and reverse in my observation.
Way 2: Switch to "Agree" and "Disagree" permanently and/or reject "Values" so, that the title goes bigger. Once Anti-Centrism is reached, all Ideology-Balls are 100% contented in my experience.
Way 3: Figuring out what the Ideology-Balls individually appeals due changes in four axis.
Way 4: Making way 3 easier with using kamikaze till one ball stands.
Way 5: Use "Surveillance", "De*** Flights"/"Re-directive Institutional Facility" and "Personality Cult/Liberation Rally" till sameness among the balls are reached.
Way 6: Pressing "Meh" always till turning to "Radical Centrism" and "Reform" the country every round (except warfare) or shapeswifting.
Way 7: Depends on thinking that refusing to answer the questions fits the definition too.
governance
[noun](https://www.merriam-webster.com/dictionary/noun)
gov·​er·​nance ˈgə-vər-nən(t)s 
plural governances
: the act or process of [governing](https://www.merriam-webster.com/dictionary/govern) or overseeing the control and direction of something (such as a country or an organization)

govern
[verb](https://www.merriam-webster.com/dictionary/verb)
gov·​ern ˈgə-vərn 
governed; governing; governs

[transitive verb](https://www.merriam-webster.com/dictionary/transitive)
1
a
: to exercise continuous sovereign authority over
especially : to control and direct the making and administration of policy in
The country was governed by a king.
b
: to rule without sovereign power and usually without having the authority to determine basic policy
2
a
archaic : [manipulate](https://www.merriam-webster.com/dictionary/manipulate)
b
: to control the speed of (a machine) especially by automatic means
3
a
: to control, direct, or strongly influence the actions and conduct of
b
: to exert a determining or guiding influence in or over
income must govern expenditure
c
: to hold in check : [restrain](https://www.merriam-webster.com/dictionary/restrain)
was told to govern her emotions
4
: to require (a word) to be in a certain case
5
: to serve as a precedent or deciding principle for
customs that govern human decisions

**Reply by Fengsemul** [reply]
> Governing a country is hard, there is no one way. Go figure it out!

The long-term-apolitical way to govern: When a Polcompball/Country shapeshifts into some centrist like the Moderate-Ball or confessionalism, holds propaganda-value-slots empty, than presses Meh, than the Game-Over-screen says "Apolitical Apolitical Party" in timeline. I assume the centrist-shapeshift stops the value generation (mostly) against the Propaganda-value-generation. At least I tested it with pressing "Meh" as Apolitical, than pressing the reform-button as a Radical Apolitical.
